docs/pages/tabs.md

```diff
@@ -146,13 +146,16 @@ Add the attribute `data-active-collapse="true"` to a tabstrip to collapse active
 
 ### Browser history
 
-The current state of the tabset is now recorded by adding a hash with the tab panel ID to the browser URL when a tab is opened. By default, tabs replace the browser history. Modify this behavior by using attribute `data-update-history="true"` to *append* to the browser history. In the latter case the browser back button will track each click that opens a tab panel.
+When the `data-deep-link="true"` attribute is set, the current state of the tabset is recorded by adding a hash with the tab panel ID to the browser URL when a tab opens. By default, tabs *update and replace* the browser history (meaning that the browser does not navigate). Modify this behavior by using attribute `data-update-history="true"` to *append* to the browser history. In the latter case the browser back button will track each click that opens a tab panel.
```

Comment:
You can set options in Js, without attributes: `Foundation.tabs.defaults.updateHistory`.

There is no `history.updateState()` but `history.pushState()`. And about the difference:
```
page1 --> page2 -(pushState)-> page3 -(go back)-> page2
page1 --> page2 -(replaceState)-> page3 -(go back)-> page1
```